sdk/eventhub/azure-eventhubs/azure/eventhub/receiver.py

```diff
@@ -285,45 +290,53 @@ def receive(self, max_batch_size=None, timeout=None):
                 :caption: Receive events from the EventHub.
 
```

Comment:
Not sure if this docstring is ever made public, but we should probably add the new parameter anyway